packages/terra-table/src/TableRowContent.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {
+  /**
+   * Content to be displayed for the row cell
+   */
+  display: PropTypes.any.isRequired,
```

Comment:
Nit: Change this to `content`. It is easier to reason about what this is an matches with other react components we have.